README.md

```diff
@@ -12,6 +12,12 @@ This style guide is a compilation of [atomic components](http://bradfrost.com/bl
 - Link to the production files at `styleguide/public/assets/`
 
 ## To begin working:
+**Environment setup (mac)**
+
+ - Have [`homebrew`](https://brew.sh/) installed
+ - `brew install nvm`
```

Comment:
Or rather, I suppose I prefer what brew tells you because not every user uses `.profile` for their bash settings. I use `.bash_profile` for example and I thought the brew instructions were more broadly applicable.